client/lib/cart-values/cart-items.ts

```diff
@@ -165,6 +165,10 @@ export function hasTransferProduct( cart: ObjectWithProducts ): boolean {
 	return getAllCartItems( cart ).some( isDomainTransfer );
 }
 
+export function hasTransferProductOnly( cart: ObjectWithProducts ): boolean {
+	return getAllCartItems( cart ).every( isDomainTransfer );
+}
+
```

Comment:
This will conflict with https://github.com/Automattic/wp-calypso/pull/79920/files?w=1#diff-5ff073ae2e625628fa72d1d1bab4c0c9107de49021e65d9eab63b0402cb2bda2R171 depending on which merges first.

Reply:
Thanks for pointing this out! :pray: